rellax.js

```diff
@@ -101,34 +101,29 @@
       throw new Error("The elements you're trying to select don't exist.");
     }
 
-
     // Let's kick this script off
     // Build array for cached element values
     // Bind scroll and resize to animate method
     var init = function() {
+      for (var i = 0; i < blocks.length; i++){
+        self.elems[i].style.cssText = blocks[i].style;
+      }
+
+      blocks = [];
+
       screenY = window.innerHeight;
       screenX = window.innerWidth;
       setPosition();
 
       // Get and cache initial position of all elements
-      for (var i = 0; i < self.elems.length; i++){
+      for (i = 0; i < self.elems.length; i++){
```

Comment:
Let's keep the var in there so it scopes the i value to this block

Reply:
I removed it because JSHint complained.